src/Icons/doc/index.rst

```diff
@@ -324,17 +327,17 @@ added to all icons unless overridden by the second argument of the ``ux_icon`` f
     # config/packages/ux_icons.yaml
     ux_icons:
         default_icon_attributes:
-            fill: currentColor
+            style: "color: white;"
```

Comment:
Let's keep the documentation  as it was here (`fill: currentColor` is the default configuration value of this Icon, and I would not recommand seting a color by name on a global scope)